### ISSUE TITLE
Error: Cannot find module 'strip-ansi'

### ISSUE THREAD

**Issue by MrParano1d** [frandiox/vite-ssr#102]
Dependency 'strip-ansi' is in your master branch but not in your current release: https://github.com/frandiox/vite-ssr/releases/tag/v0.13.1

This results in an error: 

```
> vite-ssr dev
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module 'strip-ansi'
Require stack:
- F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\utils.js
- F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\server.js
- F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\index.js
- F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\utils.js:9:38)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'F:\\Development\\Projects\\Work\\hq.webapp\\node_modules\\vite-ssr\\dev\\utils.js',
    'F:\\Development\\Projects\\Work\\hq.webapp\\node_modules\\vite-ssr\\dev\\server.js',
    'F:\\Development\\Projects\\Work\\hq.webapp\\node_modules\\vite-ssr\\dev\\index.js',
    'F:\\Development\\Projects\\Work\\hq.webapp\\node_modules\\vite-ssr\\cli.js'
  ]
}
```

**Comment by MrParano1d** [comment]
And if I add it manually it returns this error:

```
> vite-ssr dev

internal/modules/cjs/loader.js:1089
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: F:\Development\Projects\Work\hq.webapp\node_modules\strip-ansi\index.js
require() of ES modules is not supported.
require() of F:\Development\Projects\Work\hq.webapp\node_modules\strip-ansi\index.js from F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\utils.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from F:\Development\Projects\Work\hq.webapp\node_modules\strip-ansi\package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1089:13)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (F:\Development\Projects\Work\hq.webapp\node_modules\vite-ssr\dev\utils.js:9:38)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by frandiox** [comment]
@MrParano1d Thanks. I've released the latest master in 0.13.2 👍
